src/runtime/components/forms/FormGroup.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+  <label :class="[ui.wrapper]">
+    <div v-if="label" :class="[ui.label.wrapper, ui.size[size]]">
+      <p :class="[ui.label.base, required ? ui.label.required : '']">{{ label }}</p>
+      <span v-if="hint" :class="[ui.hint]">{{ hint }}</span>
+    </div>
+    <p v-if="description" :class="[ui.description, ui.size[size]]">{{ description }}</p>
+    <div :class="[label ? ui.container : '']">
+      <slot v-bind="{ error: errorMessage }" />
+      <p v-if="errorMessage" :class="[ui.error, ui.size[size]]">{{ errorMessage }}</p>
+      <p v-else-if="help" :class="[ui.help, ui.size[size]]">{{ help }}</p>
+    </div>
+  </label>
+</template>
+
+<script lang="ts">
+import { computed, defineComponent, provide, inject } from 'vue'
+import type { PropType } from 'vue'
+import { defu } from 'defu'
+import type { FormError } from '../../types'
+
+// TODO: Remove
+// @ts-expect-error
+import appConfig from '#build/app.config'
+
+// const appConfig = useAppConfig()
+
+export default defineComponent({
+  props: {
+    name: {
+      type: String,
+      default: null
+    },
```

Comment:
We can depreciate this props since it's now only used for validation and rename it to `path` to make the documentation clearer.

The two can coexist by setting the validation path to `props.path ?? props.name` to avoid breaking changes

Reply:
You still need it to bind on the `<label :for="name">`

Reply:
Not if the input is wrapped into the label like I mentioned in the description

Reply:
Completely missed that sorry, it's awesome! Nice found 😊